hs_composite_resource/models.py

```diff
@@ -60,6 +60,19 @@ def get_folder_aggregation_object(self, dir_path):
                 return fl.logical_file
         return None
 
+    def get_file_aggregation_object(self, file_path):
+        """Returns an aggregation (file type) object if the specified file *file_path* represents a
+         file type aggregation (logical file), otherwise None.
+
+         :param file_path: Resource file path (full file path starting with resource id)
+         for which the aggregation object to be retrieved
+        """
+        for res_file in self.files.all():
+            if res_file.full_path == file_path:
+                if res_file.has_logical_file:
+                    return res_file.logical_file
```

Comment:
@sblack-usu combine two nested if conditions above into one with ```AND```?

Reply:
I prefer it this way... if you really want me to change it I can.

Reply:
@sblack-usu It is your call, either would work. So just go with what you prefer.